File: VitorBasso11611BCC034/programas tarefa 1/java/micro/micro02.java
public class micro02{
    public static void main(String[] args){
        int num1, num2;
        System.out.print("Digite o primeiro numero: ");
        num1 = Integer.parseInt(System.console().readLine());
        System.out.print("Digite o segundo numero: ");
        num2 = Integer.parseInt(System.console().readLine());

        if(num1 > num2){
            System.out.print("O primeiro numero " + num1 + " e maior que o segundo " + num2);
        }else{
            System.out.print("O segundo numero " + num2 + " e maior que o primeiro " + num1);
        }
    }
}

File: trabalho 7/codigos/lua/micro/micro02.lua
function main() int
    int num1, num2
    print("Digite o primeiro numero: ")
    num1 = io.read()
    print("Digite o segundo numero: ")
    num2 = io.read()

    if num1 > num2 then
        print("O primeiro número " , num1, " é maior que o segundo ", num2, " \n")
    else
        print("O segundo número ", num2, " é maior que o primeiro ", num1, " \n")
    end
    return 1
end

main()

